brunch-config.js

```diff
@@ -1,13 +1,13 @@
 // See http://brunch.io for documentation.
 exports.files = {
   javascripts: {
-    joinTo: 'app.js'
+    joinTo: 'app.js',
   },
-  stylesheets: {joinTo: 'app.css'}
+  stylesheets: { joinTo: 'app.css' },
 };
 
 exports.plugins = {
-  babel: {presets: ['latest', 'react']}
+  babel: { presets: ['latest', 'react'] },
```

Comment:
Please, try to decide on this 'latest' thing, choose a more controllable default as `latest` as too much unexpected behavior (ES7, ES8 experimental ... who knows what)